dependencies/graknlabs/repositories.bzl

```diff
@@ -36,6 +36,6 @@ def graknlabs_common():
 def graknlabs_behaviour():
     git_repository(
         name = "graknlabs_behaviour",
-        remote = "https://github.com/graknlabs/behaviour",
-        commit = "7088622b9f4a3bd99cf4fb278130ad76ff35af6b" # sync-marker: do not remove this comment, this is used for sync-dependencies by @graknlabs_behaviour
+        remote = "https://github.com/alexjpwalker/behaviour",
+        commit = "a2be18aad83740c7cd8dbce11d56da0b4872d9ca" # sync-marker: do not remove this comment, this is used for sync-dependencies by @graknlabs_behaviour
```

Comment:
https://github.com/graknlabs/behaviour/pull/138

Reply:
@alexjpwalker could you add a comment that it should be reverted when the PR you linked is merged OR make sure they are merged in correct order?